app/vmalert/README.md

```diff
@@ -479,9 +479,9 @@ or time series modification via [relabeling](https://docs.victoriametrics.com/vm
 `vmalert` runs a web-server (`-httpListenAddr`) for serving metrics and alerts endpoints:
 
 * `http://<vmalert-addr>` - UI;
-* `http://<vmalert-addr>/api/v1/rules` - list of all loaded groups and rules;
-* `http://<vmalert-addr>/api/v1/alerts` - list of all active alerts;
-* `http://<vmalert-addr>/api/v1/<groupID>/<alertID>/status"` - get alert status by ID.
+* `http://<vmalert-addr>/vmalert/api/v1/rules` - list of all loaded groups and rules;
```

Comment:
I wouldn't add `/vmalert/` prefix in the docs for backwards compatibility with Prometheus alerting API. It is OK to add `/vmalert/` prefix to all the urls generated by `vmalert`.